src/backend/gl/src/command.rs

```diff
@@ -154,6 +149,11 @@ pub enum Command {
     BindTexture(u32, n::Texture, n::TextureTarget),
     BindSampler(u32, n::Sampler),
     SetTextureSamplerSettings(u32, n::Texture, n::TextureTarget, image::SamplerDesc),
+
+    SetColorMask(Option<DrawBuffer>, pso::ColorMask),
```

Comment:
could we also use them for implementing `bind_graphics_pipeline`? this would make it a single-responsibility path

Reply:
sure!